### ISSUE TITLE
Broken field filters with a dot in the name in Vertica

### ISSUE THREAD

**Issue by grkhr** [metabase/metabase#13932]
**Describe the bug**
When I try to user field filters on columns with a dot in the name, I see error message.

**Logs**
```
[1b9f90e3-e0eb-4c25-8af0-6b485827c8cc] 2020-12-02T13:22:56+03:00 ERROR metabase.query-processor.middleware.catch-exceptions Error processing query: null
{:database_id 2,
 :started_at #t "2020-12-02T10:22:56.282690Z[GMT]",
 :via
 [{:status :failed,
   :class java.sql.SQLFeatureNotSupportedException,
   :error "[Vertica][VJDBC](2983) ERROR: Database \"XXXX\" does not exist",
   :stacktrace
   ["com.vertica.util.ServerErrorData.buildException(Unknown Source)"
    "com.vertica.io.ProtocolStream.readExpectedMessage(Unknown Source)"
    "com.vertica.dataengine.VDataEngine.prepareImpl(Unknown Source)"
    "com.vertica.dataengine.VDataEngine.prepare(Unknown Source)"
    "com.vertica.dataengine.VDataEngine.prepare(Unknown Source)"
    "com.vertica.jdbc.common.SPreparedStatement.<init>(Unknown Source)"
    "com.vertica.jdbc.jdbc4.S4PreparedStatement.<init>(Unknown Source)"
    "com.vertica.jdbc.VerticaJdbc4PreparedStatementImpl.<init>(Unknown Source)"
    "com.vertica.jdbc.VJDBCObjectFactory.createPreparedStatement(Unknown Source)"
    "com.vertica.jdbc.common.SConnection.prepareStatement(Unknown Source)"
    "com.vertica.jdbc.common.SConnection.prepareStatement(Unknown Source)"
    "com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:267)"
    "--> driver.sql_jdbc.execute$fn__76548.invokeStatic(execute.clj:238)"
    "driver.sql_jdbc.execute$fn__76548.invoke(execute.clj:236)"
    "driver.sql_jdbc.execute$prepared_statement_STAR_.invokeStatic(execute.clj:256)"
    "driver.sql_jdbc.execute$prepared_statement_STAR_.invoke(execute.clj:253)"
    "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:389)"
    "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:376)"
    "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:385)"
    "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:376)"
    "driver.sql_jdbc$fn__78126.invokeStatic(sql_jdbc.clj:49)"
    "driver.sql_jdbc$fn__78126.invoke(sql_jdbc.clj:47)"
    "query_processor.context$executef.invokeStatic(context.clj:59)"
    "query_processor.context$executef.invoke(context.clj:48)"
    "query_processor.context.default$default_runf.invokeStatic(default.clj:69)"
    "query_processor.context.default$default_runf.invoke(default.clj:67)"
    "query_processor.context$runf.invokeStatic(context.clj:45)"
    "query_processor.context$runf.invoke(context.clj:39)"
    "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
    "query_processor.reducible$pivot.invoke(reducible.clj:31)"
    "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__47457.invoke(mbql_to_native.clj:26)"
    "query_processor.middleware.check_features$check_features$fn__46732.invoke(check_features.clj:42)"
    "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__47622.invoke(optimize_datetime_filters.clj:133)"
    "query_processor.middleware.auto_parse_filter_values$auto_parse_filter_values$fn__45533.invoke(auto_parse_filter_values.clj:44)"
    "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__49188.invoke(wrap_value_literals.clj:149)"
    "query_processor.middleware.annotate$add_column_info$fn__45295.invoke(annotate.clj:575)"
    "query_processor.middleware.permissions$check_query_permissions$fn__46607.invoke(permissions.clj:70)"
    "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__48140.invoke(pre_alias_aggregations.clj:40)"
    "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__46805.invoke(cumulative_aggregations.clj:61)"
    "query_processor.middleware.resolve_joined_fields$resolve_joined_fields$fn__48377.invoke(resolve_joined_fields.clj:36)"
    "query_processor.middleware.resolve_joins$resolve_joins$fn__48696.invoke(resolve_joins.clj:183)"
    "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__41020.invoke(add_implicit_joins.clj:254)"
    "query_processor.middleware.large_int_id$convert_id_to_string$fn__47418.invoke(large_int_id.clj:44)"
    "query_processor.middleware.limit$limit$fn__47443.invoke(limit.clj:38)"
    "query_processor.middleware.format_rows$format_rows$fn__47398.invoke(format_rows.clj:84)"
    "query_processor.middleware.desugar$desugar$fn__46871.invoke(desugar.clj:22)"
    "query_processor.middleware.binning$update_binning_strategy$fn__45892.invoke(binning.clj:229)"
    "query_processor.middleware.resolve_fields$resolve_fields$fn__46409.invoke(resolve_fields.clj:24)"
    "query_processor.middleware.add_dimension_projections$add_remapping$fn__40564.invoke(add_dimension_projections.clj:316)"
    "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__40771.invoke(add_implicit_clauses.clj:141)"
    "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__41169.invoke(add_source_metadata.clj:105)"
    "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__48337.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)"
    "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__45480.invoke(auto_bucket_datetimes.clj:125)"
    "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__46456.invoke(resolve_source_table.clj:46)"
    "query_processor.middleware.parameters$substitute_parameters$fn__48122.invoke(parameters.clj:114)"
    "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__46508.invoke(resolve_referenced.clj:80)"
    "query_processor.middleware.expand_macros$expand_macros$fn__47127.invoke(expand_macros.clj:158)"
    "query_processor.middleware.add_timezone_info$add_timezone_info$fn__41200.invoke(add_timezone_info.clj:15)"
    "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__49058.invoke(splice_params_in_response.clj:32)"
    "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__48348$fn__48352.invoke(resolve_database_and_driver.clj:33)"
    "driver$do_with_driver.invokeStatic(driver.clj:61)"
    "driver$do_with_driver.invoke(driver.clj:57)"
    "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__48348.invoke(resolve_database_and_driver.clj:27)"
    "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__47345.invoke(fetch_source_query.clj:267)"
    "query_processor.middleware.store$initialize_store$fn__49067$fn__49068.invoke(store.clj:11)"
    "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
    "query_processor.store$do_with_store.invoke(store.clj:40)"
    "query_processor.middleware.store$initialize_store$fn__49067.invoke(store.clj:10)"
    "query_processor.middleware.cache$maybe_return_cached_results$fn__46385.invoke(cache.clj:214)"
    "query_processor.middleware.validate$validate_query$fn__49076.invoke(validate.clj:10)"
    "query_processor.middleware.normalize_query$normalize$fn__47470.invoke(normalize_query.clj:22)"
    "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__41038.invoke(add_rows_truncated.clj:36)"
    "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__49043.invoke(results_metadata.clj:147)"
    "query_processor.middleware.constraints$add_default_userland_constraints$fn__46748.invoke(constraints.clj:42)"
    "query_processor.middleware.process_userland_query$process_userland_query$fn__48211.invoke(process_userland_query.clj:136)"
    "query_processor.middleware.catch_exceptions$catch_exceptions$fn__46691.invoke(catch_exceptions.clj:174)"
    "query_processor.reducible$async_qp$qp_STAR___39827$thunk__39828.invoke(reducible.clj:103)"
    "query_processor.reducible$async_qp$qp_STAR___39827.invoke(reducible.clj:109)"
    "query_processor.reducible$sync_qp$qp_STAR___39836$fn__39839.invoke(reducible.clj:135)"
    "query_processor.reducible$sync_qp$qp_STAR___39836.invoke(reducible.clj:134)"
    "query_processor$process_userland_query.invokeStatic(query_processor.clj:237)"
    "query_processor$process_userland_query.doInvoke(query_processor.clj:233)"
    "query_processor$fn__49234$process_query_and_save_execution_BANG___49243$fn__49246.invoke(query_processor.clj:249)"
    "query_processor$fn__49234$process_query_and_save_execution_BANG___49243.invoke(query_processor.clj:241)"
    "query_processor$fn__49278$process_query_and_save_with_max_results_constraints_BANG___49287$fn__49290.invoke(query_processor.clj:261)"
    "query_processor$fn__49278$process_query_and_save_with_max_results_constraints_BANG___49287.invoke(query_processor.clj:254)"
    "api.dataset$fn__52603$fn__52606.invoke(dataset.clj:59)"
    "query_processor.streaming$streaming_response_STAR_$fn__37131$fn__37132.invoke(streaming.clj:73)"
    "query_processor.streaming$streaming_response_STAR_$fn__37131.invoke(streaming.clj:72)"
    "async.streaming_response$do_f_STAR_.invokeStatic(streaming_response.clj:66)"
    "async.streaming_response$do_f_STAR_.invoke(streaming_response.clj:64)"
    "async.streaming_response$do_f_async$fn__24639.invoke(streaming_response.clj:85)"],
   :state "0A000"}],
 :json_query
 {:database 2,
  :native
  {:template-tags
   {:date {:id "7182b925-39f1-34ab-f324-e15308882bab", :name "date", :display-name "Date", :type "text"},
    :result {:id "837fb05f-eeb1-f579-66db-b15538ad2cf7", :name "result", :display-name "Result value", :type "dimension", :dimension ["field-id" 7979], :widget-type "category", :default nil},
    :fail_reason {:id "80dbadfe-c2ec-f667-2732-2e815d7e6eae", :name "fail_reason", :display-name "Fail reason", :type "text"}},
   :query
   "with moder as (\r\n    select _id, manager_id, ts_to, owner_id, \"result.value\", \"result.no_answer_type\" value_result\r\n    from XXXX.moderation\r\n    where ts_to is not null\r\n        [[and {{date}}]]\r\n        [[and {{result}}]]\r\n        [[and {{fail_reason}}]]\r\n)\r\n\r\nselect * from moder\r\nlimit 20"},
  :type "native",
  :parameters [{:type "category", :target ["dimension" ["template-tag" "result"]], :value ["accomodation"]}],
  :middleware {:js-int-to-string? true, :add-default-userland-constraints? true}},
 :status :failed,
 :class com.vertica.support.exceptions.FeatureNotSupportedException,
 :stacktrace
 ["com.vertica.util.ServerErrorData.buildException(Unknown Source)"
  "com.vertica.io.ProtocolStream.readExpectedMessage(Unknown Source)"
  "com.vertica.dataengine.VDataEngine.prepareImpl(Unknown Source)"
  "com.vertica.dataengine.VDataEngine.prepare(Unknown Source)"
  "com.vertica.dataengine.VDataEngine.prepare(Unknown Source)"
  "com.vertica.jdbc.common.SPreparedStatement.<init>(Unknown Source)"
  "com.vertica.jdbc.jdbc4.S4PreparedStatement.<init>(Unknown Source)"
  "com.vertica.jdbc.VerticaJdbc4PreparedStatementImpl.<init>(Unknown Source)"
  "com.vertica.jdbc.VJDBCObjectFactory.createPreparedStatement(Unknown Source)"
  "com.vertica.jdbc.common.SConnection.prepareStatement(Unknown Source)"
  "com.vertica.jdbc.common.SConnection.prepareStatement(Unknown Source)"
  "com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:267)"
  "--> driver.sql_jdbc.execute$fn__76548.invokeStatic(execute.clj:238)"
  "driver.sql_jdbc.execute$fn__76548.invoke(execute.clj:236)"
  "driver.sql_jdbc.execute$prepared_statement_STAR_.invokeStatic(execute.clj:256)"
  "driver.sql_jdbc.execute$prepared_statement_STAR_.invoke(execute.clj:253)"
  "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:389)"
  "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:376)"
  "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:385)"
  "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:376)"
  "driver.sql_jdbc$fn__78126.invokeStatic(sql_jdbc.clj:49)"
  "driver.sql_jdbc$fn__78126.invoke(sql_jdbc.clj:47)"
  "query_processor.context$executef.invokeStatic(context.clj:59)"
  "query_processor.context$executef.invoke(context.clj:48)"
  "query_processor.context.default$default_runf.invokeStatic(default.clj:69)"
  "query_processor.context.default$default_runf.invoke(default.clj:67)"
  "query_processor.context$runf.invokeStatic(context.clj:45)"
  "query_processor.context$runf.invoke(context.clj:39)"
  "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
  "query_processor.reducible$pivot.invoke(reducible.clj:31)"
  "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__47457.invoke(mbql_to_native.clj:26)"
  "query_processor.middleware.check_features$check_features$fn__46732.invoke(check_features.clj:42)"
  "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__47622.invoke(optimize_datetime_filters.clj:133)"
  "query_processor.middleware.auto_parse_filter_values$auto_parse_filter_values$fn__45533.invoke(auto_parse_filter_values.clj:44)"
  "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__49188.invoke(wrap_value_literals.clj:149)"
  "query_processor.middleware.annotate$add_column_info$fn__45295.invoke(annotate.clj:575)"
  "query_processor.middleware.permissions$check_query_permissions$fn__46607.invoke(permissions.clj:70)"
  "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__48140.invoke(pre_alias_aggregations.clj:40)"
  "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__46805.invoke(cumulative_aggregations.clj:61)"
  "query_processor.middleware.resolve_joined_fields$resolve_joined_fields$fn__48377.invoke(resolve_joined_fields.clj:36)"
  "query_processor.middleware.resolve_joins$resolve_joins$fn__48696.invoke(resolve_joins.clj:183)"
  "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__41020.invoke(add_implicit_joins.clj:254)"
  "query_processor.middleware.large_int_id$convert_id_to_string$fn__47418.invoke(large_int_id.clj:44)"
  "query_processor.middleware.limit$limit$fn__47443.invoke(limit.clj:38)"
  "query_processor.middleware.format_rows$format_rows$fn__47398.invoke(format_rows.clj:84)"
  "query_processor.middleware.desugar$desugar$fn__46871.invoke(desugar.clj:22)"
  "query_processor.middleware.binning$update_binning_strategy$fn__45892.invoke(binning.clj:229)"
  "query_processor.middleware.resolve_fields$resolve_fields$fn__46409.invoke(resolve_fields.clj:24)"
  "query_processor.middleware.add_dimension_projections$add_remapping$fn__40564.invoke(add_dimension_projections.clj:316)"
  "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__40771.invoke(add_implicit_clauses.clj:141)"
  "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__41169.invoke(add_source_metadata.clj:105)"
  "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__48337.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)"
  "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__45480.invoke(auto_bucket_datetimes.clj:125)"
  "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__46456.invoke(resolve_source_table.clj:46)"
  "query_processor.middleware.parameters$substitute_parameters$fn__48122.invoke(parameters.clj:114)"
  "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__46508.invoke(resolve_referenced.clj:80)"
  "query_processor.middleware.expand_macros$expand_macros$fn__47127.invoke(expand_macros.clj:158)"
  "query_processor.middleware.add_timezone_info$add_timezone_info$fn__41200.invoke(add_timezone_info.clj:15)"
  "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__49058.invoke(splice_params_in_response.clj:32)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__48348$fn__48352.invoke(resolve_database_and_driver.clj:33)"
  "driver$do_with_driver.invokeStatic(driver.clj:61)"
  "driver$do_with_driver.invoke(driver.clj:57)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__48348.invoke(resolve_database_and_driver.clj:27)"
  "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__47345.invoke(fetch_source_query.clj:267)"
  "query_processor.middleware.store$initialize_store$fn__49067$fn__49068.invoke(store.clj:11)"
  "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
  "query_processor.store$do_with_store.invoke(store.clj:40)"
  "query_processor.middleware.store$initialize_store$fn__49067.invoke(store.clj:10)"
  "query_processor.middleware.cache$maybe_return_cached_results$fn__46385.invoke(cache.clj:214)"
  "query_processor.middleware.validate$validate_query$fn__49076.invoke(validate.clj:10)"
  "query_processor.middleware.normalize_query$normalize$fn__47470.invoke(normalize_query.clj:22)"
  "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__41038.invoke(add_rows_truncated.clj:36)"
  "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__49043.invoke(results_metadata.clj:147)"
  "query_processor.middleware.constraints$add_default_userland_constraints$fn__46748.invoke(constraints.clj:42)"
  "query_processor.middleware.process_userland_query$process_userland_query$fn__48211.invoke(process_userland_query.clj:136)"
  "query_processor.middleware.catch_exceptions$catch_exceptions$fn__46691.invoke(catch_exceptions.clj:174)"
  "query_processor.reducible$async_qp$qp_STAR___39827$thunk__39828.invoke(reducible.clj:103)"
  "query_processor.reducible$async_qp$qp_STAR___39827.invoke(reducible.clj:109)"
  "query_processor.reducible$sync_qp$qp_STAR___39836$fn__39839.invoke(reducible.clj:135)"
  "query_processor.reducible$sync_qp$qp_STAR___39836.invoke(reducible.clj:134)"
  "query_processor$process_userland_query.invokeStatic(query_processor.clj:237)"
  "query_processor$process_userland_query.doInvoke(query_processor.clj:233)"
  "query_processor$fn__49234$process_query_and_save_execution_BANG___49243$fn__49246.invoke(query_processor.clj:249)"
  "query_processor$fn__49234$process_query_and_save_execution_BANG___49243.invoke(query_processor.clj:241)"
  "query_processor$fn__49278$process_query_and_save_with_max_results_constraints_BANG___49287$fn__49290.invoke(query_processor.clj:261)"
  "query_processor$fn__49278$process_query_and_save_with_max_results_constraints_BANG___49287.invoke(query_processor.clj:254)"
  "api.dataset$fn__52603$fn__52606.invoke(dataset.clj:59)"
  "query_processor.streaming$streaming_response_STAR_$fn__37131$fn__37132.invoke(streaming.clj:73)"
  "query_processor.streaming$streaming_response_STAR_$fn__37131.invoke(streaming.clj:72)"
  "async.streaming_response$do_f_STAR_.invokeStatic(streaming_response.clj:66)"
  "async.streaming_response$do_f_STAR_.invoke(streaming_response.clj:64)"
  "async.streaming_response$do_f_async$fn__24639.invoke(streaming_response.clj:85)"],
 :context :ad-hoc,
 :error "[Vertica][VJDBC](2983) ERROR: Database \"XXXX\" does not exist",
 :row_count 0,
 :running_time 0,
 :data {:rows [], :cols []}}
```

**To Reproduce**
1. Make table with category column and a dot in the name, e.g. "result.value" in Vertica
2. Try to use field filter in SQL query on it
3. See error

**Expected behavior**
It should work %(

**Screenshots**
![image](https://user-images.githubusercontent.com/40689054/100860904-2a107500-34a2-11eb-8188-c9951e06ecf8.png)

**Severity**
Not extremely severe, but very uncomfortable, because we need to crutch it with a view.

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "en",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.7+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.7",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.7+10",
    "os.name": "Linux",
    "os.version": "4.4.0-137-generic",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "bigquery",
      "mysql",
      "postgres",
      "presto",
      "vertica"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "11.2 (Debian 11.2-1.pgdg90+1)"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-11-16",
      "tag": "v0.37.2",
      "branch": "release-x.37.x",
      "hash": "25e5f70"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Comment by flamber** [comment]
@grkhr Perhaps Vertica is behaving differently than most of the other drivers, but can you see the full query that is failing on the Vertica server log?
I would expect a different error, like when dataset isn't included on BigQuery.

**Comment by grkhr** [comment]
@flamber 

Part of generated query with field filter:
`"XXXX"."moderation"."result"."no_answer_type" IN (?, ?)`
and then I get **ERROR: Database \"XXXX\" does not exist**

When I look at the table in custom question it's generating `"XXXX"."moderation"."result.no_answer_type"`

So the problem is in the wrong quotes

**Comment by grkhr** [comment]
Hi! Any updates?

**Comment by flamber** [comment]
@grkhr There's 2000 other issues, but you are more than welcome to submit a PR.

**Comment by grkhr** [comment]
Still appears in 0.38.2 :( 

If you have time by a chance would be nice to fix it, I can't do a PR. Also didn't find `(defmethod sql.qp/quote-style ...` in [vertica driver handler](https://github.com/metabase/metabase/blob/aa211a57565b8aea4c7d2d0e38e684713f2a1f5a/modules/drivers/vertica/src/metabase/driver/vertica.clj).

<img width="1480" alt="image" src="https://user-images.githubusercontent.com/40689054/112622304-89203e00-8e33-11eb-9849-0d00adc0ca38.png">

**Comment by flamber** [comment]
@grkhr 
Until the issue is closed with a milestone, then it will likely not be fixed. There are 2000+ other issues and we're a small team.

**Comment by camsaul** [comment]
This was actually fixed by #19384

PR to add a test is at #19766

**Comment by camsaul** [comment]
Fixed by #19384
